txp/Pos/Txp/Topsort.hs

```diff
@@ -30,6 +30,7 @@ data TopsortState a = TopsortState
 $(makeLenses ''TopsortState)
 
 -- Why?
+-- ChShersh: Yes, I'm also interested, why? Also, there's 'hashNub' as well.
 {-# ANN topsortTxs ("HLint: ignore Use ordNub" :: Text) #-}
```

Comment:
@volhovm do you know? :)

Reply:
No idea, but if you can fix it, do that. We have tests for this function anyway.